### ISSUE TITLE
when no `root-dir` is set it fails

### ISSUE THREAD

**Issue by zoechi** [a14n/dart-ghpages-generator#9]
```
 (master) $ generate_ghpages --with-examples
examples compilation...
Unhandled exception:
Uncaught Error: FileSystemException: Rename failed, path = '/home/myuser/.pub-cache/global_packages/ghpages_generator/bin/../bin-ghpages-1431937828749/build/example' (OS Error: No such file or directory, errno = 2)
Stack Trace:
#0      _Directory.renameSync (dart:io/directory_impl.dart:212)
#1      Generator.generate.<anonymous closure>.<anonymous closure> (package:ghpages_generator/ghpages_generator.dart:189)
#2      _RootZone.runUnary (dart:async/zone.dart:1166)
#3      _Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:494)
#4      _Future._propagateToListeners (dart:async/future_impl.dart:577)
#5      _Future._completeWithValue (dart:async/future_impl.dart:368)
#6      Future.wait.<anonymous closure> (dart:async/future.dart:299)
#7      _RootZone.runUnary (dart:async/zone.dart:1166)
#8      _Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:494)
#9      _Future._propagateToListeners (dart:async/future_impl.dart:577)
#10     _Future._completeWithValue (dart:async/future_impl.dart:368)
#11     _Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:422)
#12     _microtaskLoop (dart:async/schedule_microtask.dart:43)
#13     _microtaskLoopEntry (dart:async/schedule_microtask.dart:52)
#14     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#15     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:149)

#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:895)
#1      _microtaskLoop (dart:async/schedule_microtask.dart:43)
#2      _microtaskLoopEntry (dart:async/schedule_microtask.dart:52)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:149)
```

what's the root-dir anyway?

**Comment by a14n** [comment]
Thanks for the report.

I used to create a `update-ghpages.dart` file at the root of the package. `pub global run` seem not well handled by my script. I will take a look.

**Comment by a14n** [comment]
Fix via 5603ad8. Available in `0.3.0`.

**Comment by zoechi** [comment]
Great, thanks a lot!
